plugins/shovel.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: shovel
+spec:
+  homepage: https://github.com/dodopizza/kubectl-shovel
+  shortDescription: Gather diagnostics for .NET Core applications
+  version: v0.1.0
+  description: |
+    Plugin for gathering diagnostics from running in Kubernetes .NET Core
+    applications
+  caveats: |
+    It can work with .NET Core 3.0+ applications and Kubernetes clusters with
+    docker runtime
```

Comment:
IIUC, the user needs admin rights on the cluster. If I'm correct, please mention this here. It doesn't hurt if you can list the required RBAC roles here.

Reply:
I've specified required permissions and mount allowance